articles/api/authentication/api-authz/_device-code.md

```diff
@@ -0,0 +1,79 @@
+# Get Device Code
+
+To begin the [Device Authorization Flow](/flows/concepts/device-auth), your application should first request a device code. 
+
+## Device Authorization Flow
+
+```http
+POST https://${account.namespace}/oauth/device/code
+Content-Type: application/x-www-form-urlencoded
+
+client_id=${account.clientId}&scope=SCOPE&audience=API_IDENTIFIER
+```
+
+```shell
+curl --request POST \
+  --url 'https://${account.namespace}/oauth/device/code' \
+  --header 'content-type: application/x-www-form-urlencoded' \
+  --data 'client_id=${account.clientId}&scope=SCOPE&audience=API_IDENTIFIER'
+```
+
+```javascript
+var request = require("request");
+
+var options = { method: 'POST',
+  url: 'https://${account.namespace}/oauth/device/code',
+  headers: { 'content-type': 'application/x-www-form-urlencoded' },
+  form:
+   { client_id: '${account.clientId}',
+     scope: 'SCOPE',
+     audience: 'API_IDENTIFIER' }
+   };
+
+request(options, function (error, response, body) {
+  if (error) throw new Error(error);
+
+  console.log(body);
+});
+```
+
+> RESPONSE SAMPLE
+
+```JSON
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
+  "device_code":"GmRh...k9eS",
+  "user_code":"WDJB-MJHT",
```

Comment:
Are we going to have a table explaining what these response values mean?

Eg:

 - device_code: Your application will use this code when polling the /oauth/token endpoint
 - user_code: The code your end user will need to fill in. You typically render this in your page asking them to go to the verification_uri and fill in the code
 - verification_uri: The URI where a user can enter the code
 - verification_uri_complete: The URI which also includes the user_code. Useful if you want to open a browser with the code already filled in or you need to generate a QR code.
 - expires_in: Expiration of the codes (in seconds)
 - interval: Polling interval in seconds for the client to call the /oauth/token endpoint

Reply:
Strictly following the format of our API ref, this info isn't typically included. I can see where it is useful, though, both generally and particularly for this flow, so I am adding it for this section. I'll add a ticket as part of our API ref rework to consider adding response values for other endpoints.